app/controllers/candidate_interface/work_history_controller.rb

```diff
@@ -0,0 +1,24 @@
+module CandidateInterface
+  class WorkHistoryController < CandidateInterfaceController
```

Comment:
Do we plan to follow the same pattern as contact details with how the controllers are set out? I.e. `WorkHistory::LengthController`, `WorkHistory::JobController`, `WorkHistory::ReviewController`. If we follow this, then this controller could be `WorkHistory::LengthController` with a Rails's `#edit` and `#update`.